src/commands/npm/monorepo/release.ts

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
I don't know if I love `monorepo`. What are the alternatives?

1. What if we did the same command but it was just smart and chose the right path (you already have error handling).
    The flags are the same, but I guess will they always be?
1. What if we did the same command with a `--multi` flag?
1. What about `npm:package:mulit:release`?
1. What about `npm:multipackage:release`?

Reply:
@amphro I think I prefer to have them as separate commands. Mostly because the behavior of the `--sign` flag is different between the two.

For the multi package use case, `--sign` is a list of packages you want signed whereas in the single package use case it's a simple boolean.

And there could easily be more differences in the future. For example, the `--npmtag` flag is currently used for every package in the lerna repo but what if we wanted to support using a specific tag for a specific package? In that case I think it would be hard to configure the flags to work for both the single and multi use cases.

Reply:
I'm okay changing the names though. Perhaps `npm:lerna:release`?

Reply:
sounds good